### PR TITLE
Add Sanity

### PR DESCRIPTION
**Pull request by vicr123**
This PR sprinkles some sanity into the Updates page and also filters for common file types in the Open dialog box.

### CODE REVIEW

**Comment by ntfluoride** [comment]
### lmao

**Comment by Royce551** [comment]
i'm dyinggggggg

**Comment by ntfluoride** [comment]
merge i dare you

File: FRESHMusicPlayer/FRESHMusicPlayer/UserInterface.cs [hunk 1]
```diff
@@ -119,6 +119,7 @@ private async void BrowseMusic()
         {
             using (var selectFileDialog = new OpenFileDialog())
             {
+                selectFileDialog.Filter = "Audio|*.mp3;*.wma;*.oga;*.ogg;*.acc;*.wav;*.flec|All Files|*";
```
Comment (Royce551) [inline FRESHMusicPlayer/FRESHMusicPlayer/UserInterface.cs]: does not show all file formats naudio supports, also "flac" is mispelled

**Review by Royce551 — CHANGES_REQUESTED** [review]

File: FRESHMusicPlayer/FRESHMusicPlayer/UserInterface.cs (hunk 1 above)
Comment (ntfluoride) [reply FRESHMusicPlayer/FRESHMusicPlayer/UserInterface.cs]: Dear Sir and/or Madam,

I believe you have too high of standards for this piece of software. Mr. Victor Tran is a very competent developer who never makes mistakes. He is six feet tall and owns five mansions.

**Comment by Royce551** [comment]
See #5

**Pull request closed without merging**